File: Iniciante/1066 - Pares, Ímpares, Positivos e Negativos/Python/1066.py
def main():
    lista = []
    for i in range(5):
        entrada = int(input())
        lista.append(entrada)

    verificador(lista)


def verificador(l):
    contador_par = 0
    contador_impar = 0
    contador_positivo = 0
    contador_negativo = 0

    for i in l:
        if i % 2 == 0:
            contador_par += 1
        elif i % 2 != 0:
            contador_impar += 1

        if i > 0:
            contador_positivo += 1
        elif i < 0 and i != 0:
            contador_negativo += 1

    print(f"{contador_par} valor(es) par(es)")
    print(f"{contador_impar} valor(es) impar(es)")
    print(f"{contador_positivo} valor(es) positivo(s)")
    print(f"{contador_negativo} valor(es) negativo(s)")


main()

File: Iniciante/1066 - Pares, Ímpares, Positivos e Negativos/JavaScript/1066.js
function main() {
  var input = require("fs").readFileSync("/dev/stdin", "utf8");
  var lines = input.split("\n");

  entradas = lines.map((item) => parseInt(item, 10));

  verificador(entradas);
}

function verificador(l) {
  let contadorPar = 0;
  let contadorImpar = 0;
  let contadorPositivo = 0;
  let contadorNegativo = 0;

  for (let i of l) {
    if (i % 2 == 0) {
      contadorPar++;
    } else if (i % 2 === 1 || i % 2 === -1) {
      contadorImpar++;
    }

    if (i > 0) {
      contadorPositivo++;
    } else if (i < 0 && i !== 0) {
      contadorNegativo++;
    }
  }

  console.log(`${contadorPar} valor(es) par(es)`);
  console.log(`${contadorImpar} valor(es) impar(es)`);
  console.log(`${contadorPositivo} valor(es) positivo(s)`);
  console.log(`${contadorNegativo} valor(es) negativo(s)`);
}

main();
